### PR TITLE
chore(merge): update master with all the carbon charts fixes

### PR DESCRIPTION
**Pull request by scottdickerson**
puts the master branch in sync with the carbon chart fixes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,8 +63,9 @@
     ]
   },
   "dependencies": {
-    "@carbon/charts": "0.16.17",
-    "@carbon/charts-react": "0.16.17",
+    "@carbon/charts": "https://github.com/scottdickerson/carbon-charts#carbon-charts-v0.16.26-gitpkg",
```
Comment (tay1orjones) [inline package.json]: What's going on here? Contribution/PR that hasn't been merged yet in the main repo?

Comment (scottdickerson) [reply package.json]: yeah exactly.  Still waiting on PRs to be merged

**Comment by davidicus** [comment]
Did we mean to bring the imagehotspot component story back to the storybook? I think Bryan removed that recently.

File: babel.config.js [hunk 2]
```diff
@@ -1,7 +1,20 @@
 module.exports = function generateConfig(api) {
   api.cache(true);
   return {
-    presets: [['@babel/preset-env'], '@babel/preset-react', '@babel/preset-flow'],
+    presets: [
+      [
+        '@babel/preset-env',
+        {
+          targets: {
+            browsers: ['last 1 version', 'ie >= 11'],
```
Comment (davidicus) [inline babel.config.js]: Isn't the default the last 2 versions? Do we want to lower the browser coverage here?

Comment (scottdickerson) [reply babel.config.js]: cool, will fix

**Comment by scottdickerson** [comment]
@davidicus yeah bringing back the ImageHotspots story was intentional, there was good stuff to snapshot in there..

**Comment by scottdickerson** [comment]
@tay1orjones @davidicus any chance of approval?

**Comment by scottdickerson** [comment]
@tay1orjones @bryancboyd I need one more review please to merge

**Comment by bryancboyd** [comment]
> @davidicus yeah bringing back the ImageHotspots story was intentional, there was good stuff to snapshot in there..

@scottdickerson let's snapshot test but hide the story from storybook... we don't export the component anyway.

**Pull request merged**